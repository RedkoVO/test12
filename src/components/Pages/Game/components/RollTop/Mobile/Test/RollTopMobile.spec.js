import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import RollTopMobile from '../'

it('renders correctly Mobile', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <RollTopMobile />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
