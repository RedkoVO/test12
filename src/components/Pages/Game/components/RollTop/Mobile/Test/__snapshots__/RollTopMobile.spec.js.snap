// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Mobile 1`] = `
<div
  className="RollTopMobile-root-1"
>
  <div
    className="RollTopMobile-gameRollInf-2"
  >
    <div
      className="HeaderGame-root-3"
    >
      <div
        className="HeaderGame-gameUserBlock-4"
      >
        <p
          className="HeaderGame-gameUserHello-5"
        >
          Hello, Ben A 
          <br />
          <a
            className="HeaderGame-gameUserVerification-6"
            href="/"
            onClick={[Function]}
          >
            KYC VERIFICATION
          </a>
        </p>
        <div
          className="HeaderGame-gameUserPhoto-7"
        >
          <img
            alt=""
            src="user.jpg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
